File: src/main/Java/boj_step2/boj9498.java
import java.io.*;

public class boj9498 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        int score = Integer.parseInt(reader.readLine());
        String grade;
        switch (score / 10) {
            case 10:
                grade = "A";
                break;
            case 9:
                grade = "A";
                break;
            case 8:
                grade = "B";
                break;
            case 7:
                grade = "C";
                break;
            case 6:
                grade = "D";
                break;
            default:
                grade = "F";
                break;
        }
        writer.append(grade);
        writer.close();
    }
}

File: src/main/Kotlin/others/boj9498.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val score = r.readLine().toInt()
    val grade = if( score >= 90) "A" else if(score >= 80) "B" else if(score >= 70) "C" else if(score >= 60) "D" else "F"
    w.write(grade)
    w.close()
}